### DISCUSSION TITLE
It should be possible to disable pull requests

### DISCUSSION THREAD

**Discussion by yurivict** [community/community#20346] [Pull Requests]
Some projects, like FreeBSD, use GitGub only as a mirror, and they have their own ways to submit patches.

But GitHub doesn’t allow to disable pull requests, and users who submit pull requests anyway have their contributions delayed or lost because of this.

See the downstream bug report that can't be resolved without such GH feature: https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=265115

(forwarded from https://github.community/t/it-should-be-possible-to-disable-pull-requests/261264)

**Comment by grahamperrin** [comment]
Duplicate of https://github.com/github-community/community/discussions/8907
